src/snippets/interview.v1.yaml

```diff
@@ -20,6 +20,13 @@ properties:
     updated:
         title: Updated date (UTC)
         $ref: ../misc/date-time.v1.yaml
+    image:
+        type: object
+        properties:
+            thumbnail:
+                $ref: ../misc/image.v1.yaml
+        required:
+          - thumbnail
```

Comment:
Hmm, since `thumbnail` is overall optional there's no need to require `image` to contain a value if present. But, it doesn't make much of a difference and it already like this elsewhere, so let's leave it.